module-apps/apps-common/popups/BluetoothAuthenticatePopup.cpp

```diff
@@ -81,7 +81,7 @@ namespace gui
         inputText->drawUnderline(false);
         inputText->setTextType(TextType::SingleLine);
         inputText->setTextEllipsisType(TextEllipsis::Both);
-        inputText->setFont(style::window::font::largelight);
+        inputText->setFont(Fonts::LargeLight);
```

Comment:
all these renames actually add nothing. 
why can't you just change past constexprs:
`style::window::font ....` to the enum you've added?

It will result in much less code change, with literally no difference in these lines. And mass renames result only in less readable PR :(

Reply:
reverted all changes to setFont-like methods
also taking in mind:
https://github.com/mudita/MuditaOS/pull/3649#discussion_r829848895
removing Fonts enum and replacing it with string to string mapping (typename to filename ie "medium" -> "gt_pressura_99")